### PR TITLE
Katello 3.18 config

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: configs/katello/3.18.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    :redmine_version_id: 1272
+:prior_releases:
+  :3.17.0:
+    :redmine_version_id: 1282
+  :3.16.2:
+    :redmine_version_id: 1308
+:code_name:
+:mash_scripts:
+  - katello-3.18-rhel7
+  - katello-pulpcore-3.18-el7
```
Comment (evgeni) [inline configs/katello/3.18.yaml:15]:
```suggestion
```

pulpcore is a standalone repo now

File: configs/katello/3.18.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      katello-pulpcore-3.18-el7-build:
+        katello-pulpcore-3.18-el7-override: 0
+      katello-pulpcore-3.18-el7-override:
+        katello-pulpcore-3.18-el7: 0
+    build_package_group_source_tag: katello-pulpcore-nightly-el7-build
+    external_repos:
+      - centos7-os
+      - centos7-updates
+      - centos-sclo-rh-rhel-7
+      - epel-7
```
Comment (evgeni) [inline configs/katello/3.18.yaml:85]:
```suggestion
```

File: configs/katello/3.18.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      katello-3.18-rhel7-override: null
+    build_target: katello-3.18-rhel7-build
+    build_package_group_source_tag: katello-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.18-rhel7-build:
+        katello-3.18-rhel7-override: 0
+        foreman-plugins-2.1-rhel7-override: 3
+        foreman-2.1-rhel7: 10
```
Comment (evgeni) [inline configs/katello/3.18.yaml:57]:
```suggestion
        foreman-2.3-rhel7: 10
```

File: configs/katello/3.18.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      katello-3.18-rhel7-override:
+        katello-3.18-rhel7: 0
+      katello-3.18-rhel7:
+        katello-3.18-thirdparty-rhel7: 0
+        foreman-2.1-nonscl-rhel7: 1
```
Comment (evgeni) [inline configs/katello/3.18.yaml:62]:
```suggestion
        foreman-2.3-nonscl-rhel7: 1
```

File: configs/katello/3.18.yaml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      katello-3.18-thirdparty-rhel7: katello-thirdparty-rhel7
+      katello-3.18-rhel7-override: null
+    build_target: katello-3.18-rhel7-build
+    build_package_group_source_tag: katello-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.18-rhel7-build:
+        katello-3.18-rhel7-override: 0
+        foreman-plugins-2.1-rhel7-override: 3
```
Comment (evgeni) [inline configs/katello/3.18.yaml:56]:
```suggestion
        foreman-plugins-2.3-rhel7-override: 3
```

**Comment by evgeni** [comment]
(I did execute the adjusted version of this to get koji things running)

**Comment by jturel** [comment]
@evgeni updated with your suggestions

**Review by evgeni — APPROVED** [review]

**Pull request merged**